GCMessengerSDKSample/app/src/main/res/layout/fragment_chat_form.xml

```diff
@@ -86,38 +86,31 @@
 
             <Button
                 android:id="@+id/chat_availability"
-                style="@style/submit_view"
+                style="@style/SubmitButtonStyle"
```

Comment:
Is it legit to turn this into a camelCase? 
@SzaszaGenesys is there any coding style for this?

Reply:
I have seen in the UI SDK that there are both kind of naming for styles. I've decided to use the same case (Pascal) as Google uses for styles

Reply:
We don't have coding standards. I guess camel case is good to use.